src/standalone/Vegetation/PlantHydraulics.jl

```diff
@@ -370,6 +371,15 @@ function water_flux(z1::FT, z2::FT, ψ1::FT, ψ2::FT, K1::FT, K2::FT) where {FT}
     return flux # (m/s)
 end
 
+"""
+    root_distribution(z::FT, rooting_depth::FT)
+
+Compute value of rooting probabilty density function at `z`
+"""
+function root_distribution(z::FT, rooting_depth::FT) where {FT <: AbstractFloat}
+    return FT(1.0 / rooting_depth) * exp(z / FT(rooting_depth)) # 1/m
```

Comment:
If `z` and `rooting_depth` are already FT, and we change 1.0 -> 1, do we still need the FTs in the formula?